### PR TITLE
Angular upgrade

### PR DESCRIPTION
**Pull request by jameshe42**
this is the first iteration of the project, donor section still needs links, and footer is a bit sparse. design of background can be up for change as well.

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Looks like this on iphone X

image

**Comment by prestonvanloon** [comment]
Also, I've added the yarn lock. This is better for reproducible builds. You can use `yarn` in place of `npm` and it works all the same. Install `yarn` with `npm i -g yarn`, if you don't have it.

**Comment by prestonvanloon** [comment]
Actually, on a real device it doesn't look so bad 
image

**Comment by rauljordan** [comment]
Hey @jameshe42 another thing I noticed is there is a weird lag and choppiness to the SVG parallax scroll. Is there a way to make it smoother?

**Comment by jameshe42** [comment]
> Hey @jameshe42 another thing I noticed is there is a weird lag and choppiness to the SVG parallax scroll. Is there a way to make it smoother?

good question, it might be from the size of the svg, it's about 1.1megs right now which is pretty large for an svg, there are probably things that can be cut down from it... not sure at what size it will be totally smooth though

**Comment by jameshe42** [comment]
alright i fixed the text from looking strange at half page

**Comment by rauljordan** [comment]
Reviewed and this looks great! I'm happy with closing the bounty now and adding smaller fixes later. Thoughts @prestonvanloon @terenc3t @nisdas?

**Comment by terenc3t** [comment]
Looks good to me!

**Comment by nisdas** [comment]
Looks good to me !

File: src/app/pages/home/prysmatic-home.page.component.html [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+                furthering our goal to scale
+                the Ethereum blockchain for all. We have set up a multisig smart contract to accept donations.
+            </p>
+            <div class="flex justify-center p-3">
+                <div class="field has-addons ">
+                    <p class="control">
+                        <span class="button is-static is-large">Address: </span>
+                    </p>
+                    <p class="control">
+                        <input class="input is-large" type="text" value="0x9B984D5a03980D8dc0a24506c968465424c81DbE" />
```
Comment (prestonvanloon) [inline src/app/pages/home/prysmatic-home.page.component.html]: This MUST be immutable. Please do not use an input field to display the wallet address.

File: src/app/pages/home/prysmatic-home.page.component.html [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    <div class="container bg-grey-light ">
+        <h1 class="title">Sponsors and Grants</h1>
+        <h2 class="subtitle">
+            Thank you to all who have contributed to Prysmatic Labs; we are eternally grateful.
+        </h2>
+
+        <div class="columns m-auto is-multiline is-mobile">
+            <div class="column text-center is-half-mobile ">
+                <a href="https://www.aragon.one" alt="argon-one">
+                    <img src="./assets/sponsors/aragonnest.svg" alt="aragonnest" width="100px" />
```
Comment (prestonvanloon) [inline src/app/pages/home/prysmatic-home.page.component.html]: Please do not use inline styles like "width". This is trivially done via CSS selector 
```css
.column a img {
  width: 100px;
}
```

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Prysmaticlabs</title>
+  <!-- <base href="/"> -->
```
Comment (prestonvanloon) [inline src/index.html]: Is this intentionally commented out?

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <script>document.write('<base href="' + document.location + '" />');</script>
```
Comment (prestonvanloon) [inline src/index.html]: What's the purpose of this?

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Prysmaticlabs</title>
```
Comment (prestonvanloon) [inline src/index.html]: `Prysmatic Labs`

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+  <!-- <base href="/"> -->
+  <script>document.write('<base href="' + document.location + '" />');</script>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+</head>
+<body>
+  <app-root></app-root>
+</body>
```
Comment (prestonvanloon) [inline src/index.html]: Please keep the Google Analytics tag before the end of the body.

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-53062950-11"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-53062950-11');
</script>

```

File: src/app/pages/prysmatic-master.page.component.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    </div>
+  </div>
+</nav>
+<main>
+  <router-outlet></router-outlet>
+</main>
+<footer class="footer bg-black">
+  <div class="container text-white flex is-mobile">
+    <span class="flex is-hidden-touch">
+      <a class="navbar-item" href="http://github.com/prysmaticlabs/geth-sharding">
```
Comment (prestonvanloon) [inline src/app/pages/prysmatic-master.page.component.html]: Please change this to github.com/prysmaticlabs/prysm

File: src/app/pages/prysmatic-master.page.component.html [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+        <span>
+          <fa-icon [icon]="faDiscord"></fa-icon> Discord
+        </span>
+      </a>
+    </span>
+
+  </div>
+  <div class="container text-white ">
+    <div class="mt-auto">
+      © Copyright 2018 Prysmatic Labs
```
Comment (prestonvanloon) [inline src/app/pages/prysmatic-master.page.component.html]: 2019

File: src/app/pages/prysmatic-master.page.component.html [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+      <a role="button" class="navbar-burger" data-target="navMenu" aria-label="menu" aria-expanded="false" (click)="toggleMenu()"
+        [ngClass]="menuStatus ? 'is-active':''">
+        <span aria-hidden="true"></span>
+        <span aria-hidden="true"></span>
+        <span aria-hidden="true"></span>
+      </a>
+    </div>
+    <div class="navbar-menu" id="navMenu" [ngClass]="menuStatus ? 'is-active':''">
+      <div class="navbar-start">
+        <a class="navbar-item" href="http://github.com/prysmaticlabs/geth-sharding">
```
Comment (prestonvanloon) [inline src/app/pages/prysmatic-master.page.component.html]: Please change to github.com/prysmaticlabs/prysm

File: src/app/pages/prysmatic-master.page.component.html [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+          <span>
+            <fa-icon [icon]="faMedium"></fa-icon> Blog
+          </span>
+        </a>
+        <a class="navbar-item" href="https://twitter.com/prylabs">
+          <span>
+            <fa-icon [icon]="faTwitter"></fa-icon> Twitter
+          </span>
+        </a>
+        <a class="navbar-item" href="http://gitter.im/prysmaticlabs/geth-sharding?source=orgpage">
```
Comment (prestonvanloon) [inline src/app/pages/prysmatic-master.page.component.html]: Please change to gitter.im/prysmaticlabs/prysm

File: src/app/pages/home/prysmatic-home.page.component.html [hunk 11]
```diff
@@ -0,0 +1,371 @@
...
+                <h2 class="title p-3 pink-border bg-grey-lightest rounded-t-lg">
+                    <fa-icon class="pink-text" [icon]="faComments"></fa-icon> Follow Us
+                </h2>
+                <div class="content is-size-5">
+                    <a class="navbar-item" href="https://twitter.com/prylabs">
+                        <span>
+                            <fa-icon class="pr-6" [icon]="faTwitter"></fa-icon> Twitter
+                        </span>
+                    </a>
+                    <a class="navbar-item" href="http://gitter.im/prysmaticlabs/geth-sharding?source=orgpage">
```
Comment (prestonvanloon) [inline src/app/pages/home/prysmatic-home.page.component.html]: Please change to gitter.im/prysmaticlabs/prysm

File: src/app/pages/prysmatic-master.page.component.html [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+        <span>
+          <fa-icon [icon]="faMedium"></fa-icon> Blog
+        </span>
+      </a>
+      <a class="navbar-item" href="https://twitter.com/prylabs">
+        <span>
+          <fa-icon [icon]="faTwitter"></fa-icon> Twitter
+        </span>
+      </a>
+      <a class="navbar-item" href="http://gitter.im/prysmaticlabs/geth-sharding?source=orgpage">
```
Comment (prestonvanloon) [inline src/app/pages/prysmatic-master.page.component.html]: Please change to http://gitter.im/prysmaticlabs/prysm

**Comment by jameshe42** [comment]
oh missed seeing this. i'll fix now.

File: src/app/pages/home/prysmatic-home.page.component.html (hunk 1 above)
Comment (jameshe42) [reply src/app/pages/home/prysmatic-home.page.component.html]: absolutely right, i was playing around with the design and totally missed replacing the input with a label.

**Pull request merged**